helm/mds/templates/cronjobsource.yaml

```diff
@@ -1,17 +1,17 @@
-{{- range $name, $processor := .Values.processors }}
-{{- if and $processor.enabled $processor.triggeredBy.cron }}
-apiVersion: sources.eventing.knative.dev/v1alpha1
-kind: CronJobSource
-metadata:
-  name: {{ $name }}-cronjobsource
-  namespace: {{ $.Release.Namespace }}
-spec:
-  schedule: {{ $processor.triggeredBy.cron | quote }}
-  sink:
-    ref:
-      apiVersion: v1
-      kind: Service
-      name: {{ $name }}
----
-{{- end }}
-{{- end }}
\ No newline at end of file
+# {{- range $name, $processor := .Values.processors }}
+# {{- if and $processor.enabled $processor.triggeredBy.cron }}
+# apiVersion: sources.eventing.knative.dev/v1alpha1
+# kind: CronJobSource
+# metadata:
+#   name: {{ $name }}-cronjobsource
+#   namespace: {{ $.Release.Namespace }}
+# spec:
+#   schedule: {{ $processor.triggeredBy.cron | quote }}
+#   sink:
+#     ref:
+#       apiVersion: v1
+#       kind: Service
+#       name: {{ $name }}
+# ---
+# {{- end }}
+# {{- end }}
```

Comment:
@jwtodd this is for knative eventing implementation, no?  Can it be removed?

Reply:
will do